examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -629,7 +639,8 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
     }
     // load geoJson
     function loadGeoJsonFiles(onEachFeature) {
-      $http.get("assets/data/state.json")
+      if ($scope.polygons.statePolygons == null){
```

Comment:
better to check this way?
typeof($scope.polygons.statePolygons) === "undefined" || $scope.polygons.statePolygons == null

Reply:
I think in javascript == and === are different. (null == undefined) is true but (null === undefined) is false. I will fix it any for safety because I am not very sure about that.